src/test/scala/uk/gov/nationalarchives/scannotifications/LambdaSpecUtils.scala

```diff
@@ -0,0 +1,66 @@
+package uk.gov.nationalarchives.scannotifications
+
+import com.github.tomakehurst.wiremock.WireMockServer
+import com.github.tomakehurst.wiremock.client.WireMock.{ok, post, urlEqualTo}
+import org.scalatest.{BeforeAndAfterAll, BeforeAndAfterEach}
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.prop.TableDrivenPropertyChecks
+import uk.gov.nationalarchives.scannotifications.ScanResultDecoder.{ScanDetail, ScanEvent, ScanFindingCounts}
+
+class LambdaSpecUtils extends AnyFlatSpec with Matchers with BeforeAndAfterAll with BeforeAndAfterEach with TableDrivenPropertyChecks {
+
+  def getCounts(scanEvent: ScanEvent): (Int, Int, Int, Int) = {
+    val critical = scanEvent.detail.findingSeverityCounts.critical.getOrElse(0)
+    val high = scanEvent.detail.findingSeverityCounts.high.getOrElse(0)
+    val medium = scanEvent.detail.findingSeverityCounts.medium.getOrElse(0)
+    val low = scanEvent.detail.findingSeverityCounts.low.getOrElse(0)
+    (critical, high, medium, low)
+  }
+
+  val scanEvents =
+    Table(
+      "scanEvent",
+      ScanEvent(ScanDetail("", ScanFindingCounts(Some(10), Some(100), Some(1000), Some(10000)))),
+      ScanEvent(ScanDetail("", ScanFindingCounts(Option.empty, Some(0), Option.empty, Some(10)))),
+      ScanEvent(ScanDetail("", ScanFindingCounts(Option.empty,Option.empty,Option.empty,Option.empty)))
+    )
+
+  val wiremockSesEndpoint = new WireMockServer(9001)
+
```

Comment:
Extra line breaks

Reply:
I've remove them and the other ones in the file.